pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -129,7 +130,8 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) (*apps.Deployment, error)
 		},
 		Spec: apps.DeploymentSpec{
 			Selector: &metav1.LabelSelector{
-				MatchLabels: c.getPodLabels(mgrConfig, false),
+				MatchLabels:          c.getPodLabels(mgrConfig, false),
+				RevisionHistoryLimit: opcontroller.RevisionHistoryLimit(),
```

Comment:
we need to move this to apps.Deployment.Spec, RevisionHistoryLimit is not part of selector struct https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#revision-history-limit

Reply:
@Madhu-1  wrote:
> we need to move this to apps.Deployment.Spec, RevisionHistoryLimit is not part of selector struct https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#revision-history-limit

Thanks for the hint! changed.